### PR TITLE
py-nibabel: add new version

### PR DESCRIPTION
**Pull request by ChristopherChristofi**
PyNibabel - 5.2.1: [release notes](https://github.com/nipy/nibabel/releases/tag/5.2.1)

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-nibabel/package.py [hunk 1]
```diff
@@ -13,8 +13,11 @@ class PyNibabel(PythonPackage):
     pypi = "nibabel/nibabel-3.2.1.tar.gz"
     git = "https://github.com/nipy/nibabel"
 
-    license("PDDL-1.0")
+    maintainers("ChristopherChristofi")
 
+    license("MIT")
```
Comment (bernhardkaindl) [inline var/spack/repos/builtin/packages/py-nibabel/package.py]: Thanks for adding yourself as maintainer! I've just one question regarding the license:

Besides MIT, it adds a few other Licenses for 3rd party components that it says are included:
https://github.com/nipy/nibabel/blob/master/COPYING

Could you clarify that? Is this package only installing the MIT based code and none of the 3rd party components?

Comment (ChristopherChristofi) [reply var/spack/repos/builtin/packages/py-nibabel/package.py]: Hello @bernhardkaindl, thank you for the review, yes the installation does include the 3rd party components, I have updated the license directive to reflect their respective licensing. I hope this helps

**Review by tldahlgren — COMMENTED** [review]
FYI. Confirmed the version sha256.

**Review by bernhardkaindl — APPROVED** [review]
@ChristopherChristofi! Thanks, looks good to me and built it too.

**Pull request merged**